Bender.yml

```diff
@@ -26,6 +26,8 @@ dependencies:
   spatz:              { git: git@iis-git.ee.ethz.ch:spatz/spatz.git,                    rev: 05b01d6a0045496ad82d393355158e3b3438438e } # branch: yt/hartid
   bus_err_unit:       { git: git@iis-git.ee.ethz.ch:carfield/bus_err_unit.git,          rev: 47a6436dc4b4b7f4a44f7786033b22c6d01530b2 } # branch: main
   common_cells:       { git: https://github.com/pulp-platform/common_cells.git,         version: 1.30.0                               }
+  tech_cells_generic: { git: git@github.com:pulp-platform/tech_cells_generic.git,       rev: 6f43739f37cfcb80841f1dba0408e352437c1b94 } # branch: fix/fpga_clk_mux
```

Comment:
@CyrilKoe we should make sure this branch is merged into its main :) 
Did you open a PR there?

Reply:
Ongoing https://github.com/pulp-platform/tech_cells_generic/pull/28

Reply:
Replaced with override